lib/charms/tls_certificates_interface/v4/tls_certificates.py

```diff
@@ -1064,9 +1064,17 @@ def _configure(self, _: EventBase):
     def _mode_is_valid(self, mode: Mode) -> bool:
         return mode in [Mode.UNIT, Mode.APP]
 
+    def _ensure_secret_exists(self, secret: Secret) -> None:
+        secret.get_info()
```

Comment:
I'm not sure about the naming of this method, "ensuring" that the secret exists sounds to me like we'll create it if it doesn't exist and that's not what this method does.

Reply:
https://github.com/canonical/tls-certificates-interface/pull/324/commits/934d2b01eae863bf0d8eb121e22301848e4780cf -- is this better?